src/lib/client.js

```diff
@@ -92,8 +93,7 @@ function Client(config) {
         Log: require('./log'),
         Logger: require('./logger'),
         NodesToHost: require('./nodes_to_host'),
-        Transport: require('./transport'),
-        utils: require('./utils')
```

Comment:
Technically, removing this is a breaking change? This doesn't seem to be used internally - but it might be used by plugins?

Even not removing it is probably a breaking change, as `utils` no longer includes `lodash` methods and even if it were to include them - lodash v4 methods are breaking, so I'm not sure on the best course of action here?